app/templates/sell-quick-start.pug

```diff
@@ -4,10 +4,10 @@
       .flex-row.flex-between
         .flex-center
           label.em-500(translate="ENTER_AMOUNT:")
-        .flex-center(ng-show="exchangeRate.fiat" ng-hide="status.fetching")
+        .flex-center(ng-hide="$ctrl.status.fetching")
           span.type-sm.em-500 1 BTC = {{ exchangeRate.fiat }} {{transaction.currency.code}}
           helper-button(content="EXCHANGE_RATE_HELPER")
-        .flex-row.flex-center(ng-show="status.fetching")
+        .flex-row.flex-center(ng-show="$ctrl.status.fetching")
```

Comment:
It looks like you removed status.fetching in the component https://github.com/blockchain/My-Wallet-V3-Frontend/pull/1166/files#diff-cca6532eb23453b009081d804389c9d9L76. Does this ever show?